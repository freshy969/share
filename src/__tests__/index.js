import Share from '..';

describe('Share', () => {
  test('works', () => {
    const share = Share.create({});
    expect(share.isWorking).toBe(true);
  });
});
