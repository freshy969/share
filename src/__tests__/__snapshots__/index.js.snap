// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Share networks 1`] = `
Object {
  "all": Object {
    "countsMap": Object {},
  },
  "email": Object {
    "countsMap": Object {},
  },
  "facebook": Object {
    "countsMap": Object {},
  },
  "googlePlus": Object {
    "countsMap": Object {},
  },
  "linkedin": Object {
    "countsMap": Object {},
  },
  "twitter": Object {
    "countsMap": Object {},
  },
  "whatsapp": Object {
    "countsMap": Object {},
  },
}
`;

exports[`Share networks 2`] = `
Array [
  "Facebook",
  "GooglePlus",
  "Linkedin",
  "Twitter",
  "Whatsapp",
  "Email",
]
`;
